README.md

```diff
@@ -31,4 +31,6 @@ https://welcome.realdevsquad.com/code-of-conduct.html
 - `npm run check` checks the formatting of all the files in the project. This script runs in a pre-commit hook. It will warn about the improper formatting present in the file.
 - `npm run fix` can be run for fixing the formatting before commiting your changes or to solve the formatting errors one may get while commiting the changes.
 
+### Note ⚠️ : To avoid any check failures, do run these scripts before raising the PR.
```

Comment:
```
Please make sure to run `npm run fix` before committing your files. Otherwise if there are any formatting issues in your code, GitHub checks will fail on your PR (pull request). You will then have to run the `fix` script to fix those issues and push the formatted code again and now the checks should pass.
```

Reply:
Better to have more details? 😉

Reply:
Yes😁, adding this right now.